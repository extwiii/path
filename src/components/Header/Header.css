.Dashboard-header {
  align-items: center;
  display: flex;
  justify-content: space-between;
  flex-direction: row;
  z-index: 10;
  background: white;
  padding: 0px 45px;
  height: 66px;
}

@media (min-width: 900px) {
  .Dashboard-header {
    height: 80px;
    transform: translate3d(0px, 0px, 0px);
    z-index: 1000;
    transition: transform 0.2s ease-out 0.2s;
  }
}
