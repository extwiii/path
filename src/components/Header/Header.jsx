import React from 'react';
import Logo from '../Logo/Logo';
import './Header.css';

const Header = () => (
  <header className="Dashboard-header">
    <Logo />
    <div>Search Bar</div>
    <div>hamburger Menu</div>
  </header>
);

export default Header;
