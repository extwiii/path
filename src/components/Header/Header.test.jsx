import React from 'react';
import { shallow } from 'enzyme';

import Header from './Header';

describe('<Header />', () => {
  it('renders without crash', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
