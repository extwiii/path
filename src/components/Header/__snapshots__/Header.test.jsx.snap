// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders without crash 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Logo />,
        <div>
          Search Bar
        </div>,
        <div>
          hamburger Menu
        </div>,
      ],
      "className": "Dashboard-header",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Search Bar",
        },
        "ref": null,
        "rendered": "Search Bar",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "hamburger Menu",
        },
        "ref": null,
        "rendered": "hamburger Menu",
        "type": "div",
      },
    ],
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Logo />,
          <div>
            Search Bar
          </div>,
          <div>
            hamburger Menu
          </div>,
        ],
        "className": "Dashboard-header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Search Bar",
          },
          "ref": null,
          "rendered": "Search Bar",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "hamburger Menu",
          },
          "ref": null,
          "rendered": "hamburger Menu",
          "type": "div",
        },
      ],
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
